demo/config.js

```diff
@@ -477,6 +477,7 @@ shakaDemo.Config = class {
 
     this.addSection_(MessageIds.LANGUAGE_SECTION_HEADER, docLink)
         .addTextInput_(MessageIds.AUDIO_LANGUAGE, 'preferredAudioLanguage')
+        .addTextInput_(MessageIds.AUDIO_LANGUAGE, 'preferredAudioLabel')
```

Comment:
It is necessary to add a different message since it does not do the same thing as preferredAudioLanguage